bin/init

```diff
@@ -50,7 +52,6 @@ setup_database() {
   set -e
 }
 
-: ${VELUM_PORT:=80}
 setup_database
 bundle exec rake db:migrate
-bundle exec "rails s -b 0.0.0.0 -p $VELUM_PORT Puma"
+bundle exec "rails s -b 0.0.0.0 -p $VELUM_PORT Puma" --pid tmp/pids/server.pid
```

Comment:
The `--pid` argument goes with the `rails server` command: `bundle exec "rails s -b 0.0.0.0 -p $VELUM_PORT --pid tmp/pids/server.pid Puma"`